docs/docs/installation/gitlab.md

```
@@ -42,21 +42,36 @@
-    - In the [gitlab] section, fill in personal_access_token and shared_secret. The access token can be a personal access token, or a group or project access token.
-    - Set deployment_type to 'gitlab' in [configuration.toml](https://github.com/Khulnasoft/pr-insight/blob/main/pr_insight/settings/configuration.toml)
+3. Clone this repository:
 
-5. Create a webhook in GitLab. Set the URL to ```http[s]://<PR_INSIGHT_HOSTNAME>/webhook```. Set the secret token to the generated secret from step 2.
-In the "Trigger" section, check the ‘comments’ and ‘merge request events’ boxes.
+```
+git clone https://github.com/Khulnasoft/pr-insight.git
+```
+
+4. Prepare variables and secrets. Skip this step if you plan on settings these as environment variables when running the insight:
+  1. In the configuration file/variables:
```

Markdownlint

Expected: 0; Actual: 2
Inconsistent indentation for list items at the same level

(MD005, list-indent)